scripts/media-auth.patch

```diff
@@ -1,31 +1,32 @@
-diff --git a/data/api/client-server/content-repo.yaml b/data/api/client-server/content-repo.yaml
-index f614f9c7..9d2342e4 100644
 --- a/data/api/client-server/content-repo.yaml
```

Comment:
do we still need this patch?

Reply:
no but we haven't removed the methods yet, just deprecated so might as well.

Reply:
Yeah, but we don't send auth on those endpoints anyway now, so we can just drop this patch, I'd say.

Reply:
also dropped the functions completely